src/main/java/org/example/Main.java

```diff
@@ -50,35 +41,55 @@ public static void main(String[] args) throws Exception {
             JsonObject jsonObject = new Gson().fromJson(response.toString(), JsonObject.class);
             JsonArray results = jsonObject.getAsJsonArray("results");
 
+            Class.forName("org.postgresql.Driver");
+
             Connection jDbCconnection = DriverManager.getConnection("jdbc:postgresql://localhost:5432/dex");
-            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test (brand_name) VALUES (?)");
+            PreparedStatement statement = jDbCconnection.prepareStatement("INSERT INTO medication_test2 (brand_name, strength, dosage_form) VALUES (?,?,?)");
 
 
 
                 List<String> brandNameList = new ArrayList<>();
                 List<Medication> medications = new ArrayList<>();
                 for (JsonElement resultElement : results) {
                     JsonObject result = resultElement.getAsJsonObject();
-                    JsonArray brandNames = result.getAsJsonObject("openfda").getAsJsonArray("brand_name");
-                    if (brandNames == null) {
+                    JsonArray products = result.getAsJsonArray("products");
+                    String brandName = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("brand_name").getAsString();
+                    String strength = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonArray("active_ingredients").getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("strength").getAsString();
+                    String dosage_form = products.getAsJsonArray().get(0).getAsJsonObject().getAsJsonPrimitive("dosage_form").getAsString();
+                    if (brandName == null) {
                         continue;
```

Comment:
It looks really messy sorry, but since the results are really nested.. this is how i was able to access the different values i needed. As of right now this is horrendous and it currently doesn't handle null events. I just wanted to see if i could reach all the values on my own. Next time you see this it'll be a bunch of Optionals instead of this dump, lol

Reply:
Haha I understand but I get what's going on here. I'm very curious on how you'll use Optionals for this part.